test/tools/unified-spec-runner/unified-utils.ts

```diff
@@ -92,6 +109,22 @@ export async function topologySatisfies(
     if (!ok && skipReason == null) skipReason = `has serverless set to ${r.serverless}`;
   }
 
+  if (typeof r.csfle === 'boolean') {
+    if (r.csfle) {
+      ok &&= typeof process.env.TEST_CSFLE === 'string' && process.env.TEST_CSFLE === 'true';
+
+      if (!ok && skipReason == null) {
+        skipReason = `requires csfle but the TEST_CSFLE environment variable is not set`;
+      }
+    } else {
+      ok &&= typeof process.env.TEST_CSFLE === 'undefined';
+
+      if (!ok && skipReason == null) {
+        skipReason = `requires no csfle but the TEST_CSFLE environment variable is set`;
```

Comment:
minor, wanna do "is set to `${JSON.stringify(xx)}`" it will save someone one evergreen round trip if this case is reached.